client/components/Footer.tsx

```diff
@@ -12,20 +12,20 @@ export default function Footer() {
             style={{
                 backgroundColor:
                     pathname === '/signup'
-                        ? '#fff'
+                        ? '#2785d7ca'
                         : pathname === '/login'
-                            ? '#fff'
+                            ? '#2785d7ca'
                             : pathname === '/dashboard'
-                                ? '#fff'
-                                : '#092173',
+                                ? '#2785d7ca'
+                                : '#2785d7ca',
                 color:
                     // eslint-disable-next-line no-nested-ternary
                     pathname === '/signup'
-                        ? '#092173'
+                        ? '#fff'
```

Comment:
This always evaluates to `#fff`. Can it be simplified?

Reply:
Similar comment for `backgroundColor`

Reply:
Yes. Before the redesign, it made sense, due to the constant color changes. I will proceed and delete unnecessary  code.